tox.ini

```diff
@@ -44,14 +43,52 @@ deps=
     eth-hash[pycryptodome]
     typing1: eth-typing<2
 
-[testenv:lint]
-basepython=python3
+[common-lint]
+basepython=python
 extras=lint
+allowlist_externals=black
 commands=
     mypy -p eth_utils --config-file {toxinidir}/mypy.ini
     mypy --follow-imports=silent --ignore-missing-imports --disallow-incomplete-defs tests/type-checks/
-    flake8 {toxinidir}/eth_utils {toxinidir}/tests {toxinidir}/setup.py
-    black --check --diff {toxinidir}/eth_utils/ {toxinidir}/tests/ {toxinidir}/setup.py {toxinidir}/docs
-    pydocstyle {toxinidir}/eth_utils {toxinidir}/tests
-    pytest {posargs:tests}/functional-utils/type_inference_tests.py
-allowlist_externals=black
+    flake8 --extend-ignore=E501 {toxinidir}/eth_utils {toxinidir}/tests
```

Comment:
If you don't want to fix these `E501` errors in this PR, I think I'd rather just keep the max line length as 100 on line 21 above rather than ignoring all of the `E501`s.

Reply:
Thanks for pointing that out. It was only a few files affected anyway, removed the `extend-ignore` and fixed the line lengths.